src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -0,0 +1,107 @@
+import csv
+import dataclasses
+import io
+import json
+import re
+from dataclasses import dataclass
+
+import requests
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors import NotFound
+from databricks.sdk.service.provisioning import Workspace
+
+from databricks.labs.ucx.hive_metastore import TablesCrawler
+from databricks.labs.ucx.hive_metastore.tables import Table
+
+
+class WorkspaceInfoReader:
+    def __init__(self, ws: WorkspaceClient, folder: str):
+        self._ws = ws
+        self._folder = folder
+
+    def _current_workspace_id(self) -> int:
+        headers = self._ws.config.authenticate()
+        headers["User-Agent"] = self._ws.config.user_agent
+        response = requests.get(f"{self._ws.config.host}/api/2.0/preview/scim/v2/Me", headers=headers, timeout=10)
+        return int(response.headers.get("x-databricks-org-id"))
```

Comment:
This should be found in the ws config object somewhere.
`int()` casting operations need a lot of exception handling.
Add unit and integration testing.

Reply:
It's not there yet